include/mlir-tcp/Conversion/StablehloToTcp/StablehloToTcp.h

```diff
@@ -18,7 +19,7 @@ namespace mlir {
 
 namespace tcp {
 
-std::unique_ptr<Pass> createConvertStablehloToTcpPass();
+std::unique_ptr<OperationPass<func::FuncOp>> createConvertStablehloToTcpPass();
```

Comment:
[Unrelated to this PR] 
Fix conversion pass creators to be explicit about returning function passes (`std::unique_ptr<OperationPass<func::FuncOp>>`)